src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/summary/SummaryGroupComponent.tsx

```diff
@@ -120,8 +129,9 @@ function SummaryGroupComponent(props: ISummaryGroupComponent) {
         if (groupErrors) {
           break;
         }
+
         // eslint-disable-next-line no-loop-func
-        groupComponent.children.forEach((componentId: string) => {
+        groupChildComponents.forEach((componentId: string) => {
```

Comment:
Shouldn't the useEffect here subscribe to the groupChildComponents instead of the groupComponent?

Reply:
Yep, good catch!